backend/src/models/chat/dto/channel.dto.ts

```diff
@@ -26,7 +26,6 @@ export class ChannelDto {
   @ValidateIf((object, value) => value !== null)
   inviteList!: number[] | null;
 
```

Comment:
`@IsUrl()` 빠진 이유가 무엇인가요?
만약 에러를 뱉었기 때문이라면, `@IsUrl` 의 옵션에 대해서 조금 찾아보시면, `localhost:8080` 과 같은 케이스까지 허용하려면

```suggestion
@IsUrl({ require_tld: false })
```
 과 같은 옵션을 부여하여 허용할 수 있습니다.

Reply:
반영 했습니다~